packages/module/src/modules/config/addons/globals.ts

```diff
@@ -20,7 +20,7 @@ export function createAddonGlobals(nuxt: Nuxt): ESLintConfigGenAddon {
       const imports = [
         ...await unimport?.getImports() || [],
         ...await nitroUnimport?.getImports() || [],
-      ].sort()
+      ].sort((a, b) => 10 * a.from.localeCompare(b.from) + a.name.localeCompare(b.name))
```

Comment:
What's the `10 *` for?

Reply:
I was meaning to sort first by from, then by name.

Reply:
Oh oh I see. First time I see such a pattern, it's indeed smart!

Reply:
it might be we could just do `a.from.localeCompare(b.from) || a.name.localeCompare(b.name))` ....